Squeezer/src/main/res/layout/alarm_playlist_item.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<TextView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/label"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_weight="1"
+    android:ellipsize="end"
+    android:maxLines="1"
+    android:paddingEnd="16dp"
+    android:textAppearance="?attr/textAppearanceSubtitle1"
+    tools:text="text" />
+</LinearLayout>
```

Comment:
This doesn't compile (using gradle from the command line)

Reply:
Doesn't compile why exactly? Compiles just fine for me:
```
./gradlew assembleDebug
Configuration cache is an incubating feature.
Reusing configuration cache.

BUILD SUCCESSFUL in 5s
30 actionable tasks: 13 executed, 17 up-to-date
Configuration cache entry reused.
```

Reply:
There is no `LinearLayout` in alarm_playlist_item.xml so you can't end it. Can be seen e.g. by `./gradlew clean build`